specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -128,60 +128,60 @@ should be created only when there is a natural developer constituency
 and a well-understood class of applications that can benefit from it. It
 is also recommended that a profile cast a comprehensive net on its area
 of interest, to minimize the occurrence of overlapping or competing
-profiles. Java EE platform features such as optional components and
+profiles. Jakarta EE platform features such as optional components and
 extensibility can be used by profiles to achieve a better fit to their
 intended target.
 
 === Application Components
 
-The Java EE runtime environment defines four
-application component types that a Java EE product must support:
+The Jakarta EE runtime environment defines four
+application component types that a Jakarta EE product must support:
 
 * Application clients are Java programming
 language programs that are typically GUI programs that execute on a
 desktop computer. Application clients offer a user experience similar to
 that of native applications and have access to all of the facilities of
-the Java EE middle tier.
+the Jakarta EE middle tier.
 * Applets are GUI components that typically
 execute in a web browser, but can execute in a variety of other
 applications or devices that support the applet programming model.
-Applets can be used to provide a powerful user interface for Java EE
+Applets can be used to provide a powerful user interface for Jakarta EE
 applications. (Simple HTML pages can also be used to provide a more
-limited user interface for Java EE applications.)
-* Servlets, JSP pages, JSF applications,
+limited user interface for Jakarta EE applications.)
+* Jakarta Servlets, Jakarta Server Pages pages, Jakarta Server Faces applications,
```

Comment:
I thought JSF was renamed to "Jakarta Faces" but I can't find the final decision so I mantained it as Jakarta Server Faces. Same for JSP.

Reply:
No, it's "Jakarta Server Faces" in the list [here](https://docs.google.com/spreadsheets/d/18SraPxRBCOyaS6w-UV6TR-UA1bWy1--sV0ky6msAjWY/edit#gid=0).